lib/algrep.gi

```diff
@@ -1254,8 +1254,8 @@ end );
 
 #############################################################################
 ##
-#M  \/( <V>, <W> )  . . . . . . .factor of a an algebra module by a submodule
-#M  \/( <V>, <vectors> )  . . . .factor of a an algebra module by a submodule
+#M  \/( <V>, <W> )  . . . . . . . factor of an algebra module by a submodule
+#M  \/( <V>, <vectors> )  . . . . factor of an algebra module by a submodule
```

Comment:
The text behind the dots could be moved one more space to the right.

Reply:
Done, thanks.